src/applications/edu-benefits/components/SchoolSelectField.jsx

```diff
@@ -133,19 +146,34 @@ export class SchoolSelectField extends React.Component {
         <div>
           <div className="search-controls">
             <Element name="schoolSearch"/>
-            <input
-              onChange={this.handleSearchInputChange}
-              onKeyDown={this.handleSearchInputKeyDown}
-              type="text"
-              value={searchInputValue}/>
-            <button
-              className="search-schools-button usa-button-primary"
-              onClick={this.handleSearchClick}>
-              {'Search Schools'}
-            </button>
+            <div className="search-input">
+              <input
+                onChange={this.handleSearchInputChange}
+                onKeyDown={this.handleSearchInputKeyDown}
+                type="text"
+                value={searchInputValue}/>
+              <div
+                className={clearSearchInfoClass}>
+                {showInstitutions && <span>
+                  {`${searchResultsCount} results for ${institutionQuery}`}
+                </span>}
+                <button
+                  onClick={this.handleStartOver}
+                  className="va-button-link">
+                  Start Over
+                </button>
+              </div>
+            </div>
+            <div
+              className="search-schools">
```

Comment:
Should we name this "search-trigger" to match "search-input" above? Also, perhaps we want to keep this name generic in order to make it easier to convert into a common/shared component later.

Reply:
I like that idea. I can do some renaming of the props and keep the domain specific items in the redux and react-redux stuff.